modules/serverless-supported-channel-types.adoc

```diff
@@ -0,0 +1,14 @@
+// Module included in the following assemblies:
+//
+//  * serverless/event_workflows/serverless-channels.adoc
+
+[id="serverless-supported-channel-types_{context}"]
+= Supported channel types
+
+Currently, {ServerlessProductName} only supports `InMemoryChannel` kind channels for development use, as part of the Knative Eventing Technology Preview.
+
+The following are limitations of InMemoryChannel type channels:
+
+* No event persistence is available. If a Pod goes down, events on that Pod are lost.
```

Comment:
Based on the latest guidelines Andrea shared with the team, we'll eventually start lowercasing _pod_ unless specifically referring to it as an object. With that in mind, I'd start enforcing that new guideline here.